pkg/api/v1beta1/types.go

```diff
@@ -241,6 +241,16 @@ const (
 	PullIfNotPresent PullPolicy = "PullIfNotPresent"
 )
 
+type CapabilityType string
+
+// Capabilities represent POSIX capabilities that can be added or removed to a running container.
+type Capabilities struct {
+	// Added capabilities
+	Add []CapabilityType `json:"add,omitempty"`
```

Comment:
Specifically, Add and Drop still need descriptions, in v1beta1/types.go and in v1beta2/types.go.